[SRVCOM-1831] Serverless functions: update abstracts format (#46660)
openshift/openshift-docs: modules/serverless-creating-subscriptions-kn.adoc

:_content-type: PROCEDURE
[id="serverless-creating-subscriptions-kn_{context}"]
= Creating a subscription by using the Knative (kn) CLI

> After you have created a channel and an event sink, you can create a subscription to enable event delivery. Using the Knative (`kn`) CLI to create subscriptions provides a more streamlined and intuitive user interface than modifying YAML files directly. You can use the `kn subscription create` command as well as the appropriate flags to create a subscription by using the Knative (`kn`) CLI.

Reviewer: ```suggestion
After you have created a channel and an event sink, you can create a subscription to enable event delivery. Using the Knative (`kn`) CLI to create subscriptions provides a more streamlined and intuitive user interface than modifying YAML files directly. You can use the `kn subscription create` command with the appropriate flags to create a subscription.
```
